Cpix/ContentKey.cs

```diff
@@ -44,16 +44,16 @@ internal override void ValidateLoadedEntity(CpixDocument document)
 			ValidateEntity(document);
 
 			// We skip length check if we do not have a value for an encrypted key (it will be read-only).
-			if (IsLoadedEncryptedKey && Value != null && Value.Length != Constants.ContentKeyLengthInBytes)
-				throw new InvalidCpixDataException($"A {Constants.ContentKeyLengthInBytes}-byte value must be provided for each new content key.");
+			if (IsLoadedEncryptedKey && Value != null && !Constants.ValidContentKeyLengthsInBytes.Contains(Value.Length))
+				throw new InvalidCpixDataException($"Loaded content key must have a key value key with size from the set: {Constants.ValidContentKeyLengthsHumanReadable}.");
```

Comment:
1. Extra "key" in "have a key value key with size" (the same under ValidateNewEntity). 

2. Since the key size requirements are same for both new and loaded keys, perhaps makes sense to have unified error message? Or you felt it's useful to know (and not apparent otherwise) that this error came while loading VS creating? 
Otherwise could extract this to local "ValidateContentKeySize" method that does the throwing part and call it in both locations. Perhaps then could say just "The size of content key values, in bytes, must be one of the following" (to also explicitly it's the byte-size).

Reply:
Adjustments were made. Please take a look.